src/public/js/vendor/Inspectlet.ts

```diff
@@ -0,0 +1,32 @@
+export {};
+const wid = 812310448;
```

Comment:
It is possible that some application may not have this enabled. Plus it may vary based on environment (not 100% sure).

Kindly ensure that this is configurable and we do not inject inspectlet code if its not set.

Reply:
I am not familiar with config but can we pass **wid** as env variables like we are doing in Grafana, when the wid variable will be passed it will inject inspectlet code